### PR TITLE
Ffmpeg => 7.1 & Deps Updates

### PR DESCRIPTION
**Pull request by satmandu**
- We need to make sure all the packages that depend upon ffmpeg don't break from this.
Builds:
- [x] `x86_64`
- [x] `armv7l`
##
Tested & Working properly:
- [ ] `x86_64`
- [ ] `armv7l` 
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=ffmpeg crew update \
&& yes | crew upgrade
```

### CODE REVIEW

**Comment by uberhacker** [comment]
Wow, ambitious undertaking!

**Comment by VenomousSteam81** [comment]
ffmpeg works on arm64.

**Comment by satmandu** [comment]
Let's just merge this and see what happens. We can update packages as people have issues ... But this way packages don't keep getting built against an older ffmpeg...

File: packages/avisynthplus.rb [hunk 1]
```diff
@@ -1,37 +1,25 @@
...
-        -DBUILD_SHARED_LIBS=ON \
-        .."
-    end
-    system 'ninja -C builddir'
-  end
-
-  def self.install
-    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
-  end
+  cmake_options "-DCMAKE_BUILD_TYPE:STRING='None'"
```
Comment (uberhacker) [inline packages/avisynthplus.rb:24]: Shouldn't this be `Release`?

Comment (satmandu) [reply packages/avisynthplus.rb:24]: I copied from here:

https://gitlab.archlinux.org/archlinux/packaging/packages/avisynthplus/-/blob/main/PKGBUILD?ref_type=heads

**Review by uberhacker — APPROVED** [review]

**Pull request merged**